providers/nodebb/notification.js

```diff
@@ -373,29 +385,63 @@ exports.bindNotification = function bindNotification(forum) {
      */
     function notifyHandler(data) {
         const notification = Notification.parse(data);
-        //debug(`Notification ${notification.id}: ${notification.label} received`);
-        forum.emit('log', `Notification ${notification.id}: ${notification.label} received`);
```

Comment:
looks like you moved that log into the then handler from evalBlacklist(). was that not intended?